README.md

```diff
@@ -1013,6 +1014,12 @@ roborazzi {
 }
 ```
 
+> **Note**  
+> If you are using build.gradle instead of build.gradle.kts, you need to use the set method for each assignment, like
+> ```kotlin
+> generateComposePreviewRobolectricTests.packages.set(["com.example"])
+> ```
+
```

Comment:
In README.md, highlighting using the [!NOTE] syntax did not work.
so, I used **Note**, which is the most natural looking.  

-  README.md screenshot using [!NOTE] syntax 
<img width="1061" alt="image" src="https://github.com/user-attachments/assets/0f1215aa-32ec-41b3-9391-ebf841b865a4">

Reply:
The cause of the highlight not working seems to be this expression in the first line of the README.

```md
<!-- Generated by ./gradlew generateReadme. Do not edit this file. -->
<div name="topic_top">
```

Reply:
Sorry, I think it was confusing. How about using 

```
`generateComposePreviewRobolectricTests.packages.set(["com.example"]) `
```

instead of 

```
.```
`generateComposePreviewRobolectricTests.packages.set(["com.example"]) `
.```
```

Reply:
Sorry, I checked and the `div name="topic_top"` is the cause. I apologize for the inconvenience. I think `**Note**` could be better for now.
https://gist.github.com/takahirom/e8f63f413828e3dd004def9c2c41fef2